File: src/go/pe_0002.go
package main

func main() {
	gross := 0
	a := 1
	b := 2

	for ; a < 4e6 ; {
		if a % 2 == 0 {
			gross += a
		}
		c := a + b
		a = b
		b = c
	}

	println(gross)
}

File: src/java/pe_0002.java
public class pe_0002 {
    public static void main(String[] args) {
        int gross, a, b, c;

        gross = 0;
        a = 1;
        b = 2;
        while (a < 4e6) {
            if (a % 2 == 0) {
                gross += a;
            }
            c = a + b;
            a = b;
            b = c;
        }

        System.out.println(gross);
    }
}
